src/Surface.zig

```diff
@@ -3280,7 +3280,7 @@ fn linkAtPos(
     const mouse_mods = self.mouseModsWithCapture(self.mouse.mods);
 
     // If we have the proper modifiers set then we can check for OSC8 links.
-    if (mouse_mods.equal(input.ctrlOrSuper(.{}))) hyperlink: {
+    if (mouse_mods.ctrl or mouse_mods.super) hyperlink: {
```

Comment:
This logic isn't correct. 

We match `ctrl` on Linux and `super` on Mac. This changed this behavior.